pkg/registry/rbac/clusterrolebinding/policybased/storage.go

```diff
@@ -63,6 +66,12 @@ func (s *Storage) Update(ctx genericapirequest.Context, name string, obj rest.Up
 	nonEscalatingInfo := rest.WrapUpdatedObjectInfo(obj, func(ctx genericapirequest.Context, obj runtime.Object, oldObj runtime.Object) (runtime.Object, error) {
 		clusterRoleBinding := obj.(*rbac.ClusterRoleBinding)
 
+		// if we're explicitly authorized to bind this clusterrole, return
+		if canBind, _, _ := rbacregistry.AuthorizeBinding(ctx, clusterRoleBinding.RoleRef, clusterRoleBinding.Namespace, s.authorizer); canBind {
```

Comment:
not sure, I don't think the explicit grant will be as common as just inheriting permissions, so I'd expect this check to fail most of the time, but it's cheaper than the covers check, so I want to do it first.